src/services/ui/src/utils/crumbs.ts

```diff
@@ -1,25 +1,61 @@
-import { BreadCrumbConfig, Route, urlEmbedQuery } from "@/components";
-import { SPA_ID_REGEX } from "@/consts";
+import { BreadCrumbConfig, Route } from "@/components";
 import { mapActionLabel, mapSubmissionCrumb } from "@/utils";
-import { Action } from "shared-types";
+import { Action, Authority } from "shared-types";
 
-export const dashboardCrumb = (id?: string): BreadCrumbConfig => {
+type DetailsAndActionsBreadCrumbsArgs = {
+  id: string;
+  authority: Authority;
+  actionType?: Action;
+};
+
+export const getDashboardTabForAuthority = (
+  authority: Authority,
+): "spas" | "waivers" => {
+  switch (authority) {
+    case "CHIP SPA" as Authority:
+    case "Medicaid SPA" as Authority:
```

Comment:
This casting is due to `authority` being normal case (i.e. `Medicaid SPA`) and `Authority` only has lowercase values